src/testdir/test_gui.vim

```diff
@@ -46,6 +46,10 @@ func Test_colorscheme()
   call assert_equal('dark', &background)
```

Comment:
Unrelated to my change, but I wonder whether the above "sleep 200m"
is really necessary. At least test passes locally without it.  
Not sure why it would be needed.